score-lang-cli/src/main/java/org/openscore/lang/cli/services/ScoreServicesImpl.java

```diff
@@ -131,6 +131,37 @@ public synchronized void onEvent(ScoreEvent scoreEvent) throws InterruptedExcept
                         printWithColor(Ansi.Color.YELLOW, prefix + taskName);
                     }
                     break;
+                case ScoreLangConstants.EVENT_OUTPUT_END:
```

Comment:
I first make sure I print just the actual outputs not also the published ones. Then get the prefix (nr. of '-'s)
When printing the outputs I truncate the value if the length is greater than 30, or put "(empty)" if output has no value.
I also replace newlines and then print the actual output (key - value)